incubator/hnc/internal/validators/object_test.go

```diff
@@ -423,15 +423,45 @@ func TestUserChanges(t *testing.T) {
 			},
 		},
 	}, {
-		name: "Allow creation of object with valid selector and treeSelect annotation",
-		fail: false,
+		name: "Deny creation of object with both selector and treeSelect annotation",
+		fail: true,
+		inst: &unstructured.Unstructured{
+			Object: map[string]interface{}{
+				"apiVersion": "v1",
+				"kind":       "Pod",
+				"metadata": map[string]interface{}{
+					"annotations": map[string]interface{}{
+						api.AnnotationSelector:     "foo",
+						api.AnnotationTreeSelector: "!bar",
+					},
+				},
+			},
+		},
+	}, {
+		name: "Deny creation of object with both selector and noneSelect annotation",
+		fail: true,
```

Comment:
You don't need to test every possible combination of having multiple selectors IMO if the code itself doesn't care which ones you get.

Reply:
Done